docs/features/logforwarding/outputs/otlp-forwarding.adoc

```diff
@@ -49,98 +49,93 @@ spec:
+. `otlp` `authentication` is optional and specifies a `token.from` and a value of '*serviceAccount*'
 .. The token can also be read from a secret
 .. Also available with `username` and `password` authentication spec (refer to HTTP Auth Specification for full scope)
+. `otlp` `tuning` is optional and includes standard http tuning options in addition to `compression`: '*gzip*'.
+. `tls` includes the standard certificate configuration or specify `insecureSkipVerify`: '*true*'
 
 
 .TLS InsecureSkipVerify
 NOTE: This option is *NOT* recommended for production configurations. If true, the client will be configured to skip validating server certificates.
 
-https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/clusterlogforwarder_types.go#L244[TLS Specification] for _observability.openshift.io/v1_
-
-https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L203[HTTP Auth Specification] for _observability.openshift.io/v1_
-
-https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L812[Tuning Specification] for _observability.openshift.io/v1_
+.API Details for _observability.openshift.io/v1_
+* https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/clusterlogforwarder_types.go#L261[TLS Spec]
+* https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L242[HTTP Auth Spec]
+* https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L201[HTTP Tuning Spec]
+* https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L1225[OTLP Tuning Spec]
 
 
 == Data Model
 
 === Semantic Convention
-The Semantic Conventions in OpenTelemetry define a *Resource* as an immutable representation of the entity producing telemetry as *Attributes*.
+The Semantic Conventions in OpenTelemetry (OTel) define a *Resource* as an immutable representation of the entity producing telemetry as *Attributes*.
+
+All *Attributes* are included in the *Resource* object. For example, a process producing telemetry that is running in a container has a container_name, a cluster_id, a pod_name, a namespace, and possibly a deployment or app_name.
 
-For example, a process producing telemetry that is running in a container has a container_name, a cluster_id, a pod_name, a namespace, and possibly a deployment or app_name.
+The grouping and reducing of common resource attributes is a key aspect of sending logs via OpenTelemetry.
 
-All of these *Attributes* are included in the *Resource* object.  This grouping and reducing of common attributes is a key aspect of sending logs via OpenTelemetry.
+.Cluster Logging Reference Docs
+* https://github.com/rhobs/observability-data-model/blob/main/cluster-logging.md[OTel Data Model Draft]
```

Comment:
Consider not calling this "Draft" given at some point that implies we need to update this text.  The document to which we reference (or version  of it) can state it being draft or not